mail_contact_type/models/res_partner.py

```diff
@@ -0,0 +1,30 @@
+from odoo import fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    mail_contact_type_ids = fields.Many2many(
+        "mail.contact.type",
+        string="Mail Contact Types",
+        help="Mail Contact Types",
```

Comment:
As it it's a bit useless, I would say
```suggestion
        help="Used by email template to select contacts by mail contact type",
```